### PR TITLE
Fixes wizard academy spawner.

### LINKED ISSUE #27050: [AWAY] Wizard Academy Defender spawner runtimes and fails
```
runtime error: 
[14:16:13]Undefined operation: Academy Teacher (/mob/living/carbon/human) / 10
[14:16:13]proc name: showCandidatePollWindow (/proc/showCandidatePollWindow)
[14:16:13]  source file: game.dm,424
[14:16:13]  usr: (src)
[14:16:13]  src: null
[14:16:13]  call stack:
[14:16:13]showCandidatePollWindow(Francis Gallyus (/mob/dead/observer), Academy Teacher (/mob/living/carbon/human), "Do you want to play as Wizard ...", /list (/list), null, 37243, 1)
[14:16:13]pollCandidates("Do you want to play as Wizard ...", "wizard", null, "wizard", Academy Teacher (/mob/living/carbon/human), null, 1, /list (/list))
[14:16:13]pollGhostCandidates("Do you want to play as Wizard ...", "wizard", null, "wizard", Academy Teacher (/mob/living/carbon/human), null, 1)
[14:16:13]pollCandidatesForMob("Do you want to play as Wizard ...", "wizard", null, "wizard", Academy Teacher (/mob/living/carbon/human), null, null)
[14:16:13]Academy Defensive System (/obj/structure/academy_wizard_spawner): give control()
```

### PR DESCRIPTION
**Pull request by AnturK**
Fixes #27050

### CODE REVIEW

File: code/modules/awaymissions/mission_code/Academy.dm [hunk 1]
```diff
@@ -67,7 +67,7 @@
 	if(!current_wizard)
 		return
 	spawn(0)
```
Comment (coiax) [inline code/modules/awaymissions/mission_code/Academy.dm]: reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

Comment (AnturK) [inline code/modules/awaymissions/mission_code/Academy.dm]: Reee what ? I suppose i could make this waitfor

Comment (coiax) [inline code/modules/awaymissions/mission_code/Academy.dm]: Or use INVOKE_ASYNC

Comment (Cyberboss) [inline code/modules/awaymissions/mission_code/Academy.dm]: The guideline I follow is

Does everything calling this proc not want to wait for it's sleeps?

Y: set waitfor = FALSE
N: INVOKE_ASYNC

Comment (lzimann) [inline code/modules/awaymissions/mission_code/Academy.dm]: Just remove this spawn, it makes no sense

Comment (AnturK) [inline code/modules/awaymissions/mission_code/Academy.dm]: Oh i forgot about this PR. One moment.

File: code/modules/awaymissions/mission_code/Academy.dm [hunk 2]
```diff
@@ -64,17 +64,18 @@
 		next_check = world.time + cooldown
 
 /obj/structure/academy_wizard_spawner/proc/give_control()
+	set waitfor = 0
```
Comment (lzimann) [inline code/modules/awaymissions/mission_code/Academy.dm]: FALSE

**Pull request merged**